pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/Patch.java

```diff
@@ -161,30 +179,50 @@ public static boolean patch(@NonNull String id, @NonNull String user, @NonNull N
-            SimpleResult dbVals = new SimpleResult();
-            try (PreparedStatement stmt = conn.prepareStatement(
-                    SQL_CHECK_UNIQUE.replace(TABLE, dbElement.getTableName()).replace(COL_NAME, dbElement.getColumnName()))) {
-                try {
-                    ResultSet rs = stmt.executeQuery();
-                    if (rs.next()) {
-                        dbVals.resultValue = false;
-                    } else {
-                        dbVals.resultValue = true;
-                    }
-                } catch (Exception e) {
-                    dbVals.resultException = e;
-                }
-            } catch (SQLException ex) {
-                dbVals.resultException = ex;
-            }
-            return dbVals;
-        });
-        return (Boolean) results.resultValue;
+    static boolean isOncHistoryPatch(Patch originalPatch) {
+        return originalPatch.getNameValue().getName().equals(ONC_HISTORY_DELETED_FIELD) &&
+                originalPatch.getTableAlias().equals(DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS);
+    }
+
+    static boolean isTissuePatch(Patch originalPatch) {
+        return originalPatch.getNameValue().getName().equals(TISSUE_DELETED_FIELD) &&
+                originalPatch.getTableAlias().equals(DBConstants.DDP_TISSUE_ALIAS);
+    }
+
+    //creates a list of patches for deleting tissues that belong to an onc history
+    protected static List<Patch> getPatchForTissues(Patch oncHistoryPatch) {
+        List<String> tissueIds = TissueDao.getTissuesByOncHistoryDetailId(oncHistoryPatch.getId());
+        List<Patch> deletePatches = new ArrayList<>();
+        for (String tissueId : tissueIds) {
+            NameValue nameValue = new NameValue(TISSUE_DELETED_FIELD, TRUE_FLAG);
+            Patch patch =
+                    new Patch(tissueId, OncHistoryDetail.ONC_HISTORY_DETAIL_ID, oncHistoryPatch.getId(), oncHistoryPatch.getUser(), nameValue, null, true,
+                            oncHistoryPatch.getDdpParticipantId(), oncHistoryPatch.getRealm());
+            patch.setTableAlias(DBConstants.DDP_TISSUE_ALIAS);
+            deletePatches.add(patch);
+        }
+        return deletePatches;
+    }
+
+    //creates a list of patches for deleting the sm ids that belong to a tissue
+    protected static List<Patch> getPatchForSmIds(Patch tissuePatch) {
+        List<String> smIds = TissueSMIDDao.getSmIdPksForTissue(tissuePatch.getId());
+        List<Patch> deletePatches = new ArrayList<>();
+        for (String smIdPk : smIds) {
+            NameValue nameValue = new NameValue(SM_ID_DELETED_FIELD, TRUE_FLAG);
+            deletePatches.add(
+                    new Patch(smIdPk, TissuePatch.TISSUE_ID, tissuePatch.getId(), tissuePatch.getUser(), nameValue, null, true,
+                            tissuePatch.getDdpParticipantId(), tissuePatch.getRealm()));
+        }
+        return deletePatches;
+    }
+
+    public static boolean hasDDPParticipantId(Patch patch) {
```

Comment:
How about adding this as an instance method on `Patch`?  Having a static method that takes in an instance to inspect the instance's field is a bit of an antipattern.